scrapy/templates/project/module/middlewares.py.tmpl

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+
+# Define here the models for your spider middleware
+#
+# See documentation in:
+# http://doc.scrapy.org/en/latest/topics/spider-middleware.html
+
+
+class ${ProjectName}SpiderMiddleware(object):
+
+    def process_spider_input(response, spider):
+        # Called for each response that goes through the spider
+        # middleware and into the spider.
+
+        # Should return None or raise an exception.
+        pass
```

Comment:
I'm leaning towards providing working but no-op methods implementations.
So what about an explicit `return None` instead of `pass` explaining that it means that the response continues in the chain?